airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -303,6 +303,12 @@ def execute(self, context) -> Optional[str]:
                 "keyword project_id parameter or as project_id extra "
                 "in Google Cloud connection definition. Both are not set!"
             )
+        # There is no need to manage the kube_config file, as it will be generated automatically.
+        # All Kubernetes parameters (except config_file) are also valid for the GKEStartPodOperator.
+        if self.config_file:
+            raise AirflowException(
+                "config_file is not an allowed parameter for the GKEStartPodOperator."
+            )
```

Comment:
Could you move this to in `__init__` please? -- that way it would show up as a "DAG parse" error, rather than making the task fail at run time.

Reply:
done!